.github/workflows/bot.yml

```diff
@@ -148,3 +148,43 @@ jobs:
         uses: stefanzweifel/git-auto-commit-action@v5
         with:
           commit_message: "style: apply lint fixes"
+
+  Check_Degdratation:
+    name: Criterion Benchmarks
+    if: contains(github.event.pull_request.labels.*.name, 'benchmark')
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Install Critcmp
+        run: cargo install critcmp
+
+      - name: Run Criterion on branch
+        run: |
+          cargo bench --features criterion -- --save-baseline new_branch
+
+      - name: Fetch target branch
+        run: |
+          git fetch
+          git checkout ${{ github.sha }}
```

Comment:
`github.sha` points out to the commit that triggered the job i.e. it's the same branch that was checkouted before

Reply:
i get it will make changes accordingly